2select-html/styles.css

```
@@ -0,0 +1,11 @@
+body {
+    font-family: sans-serif;
+}
+
+h2 {
+    font-weight: normal;
+}
+
+h3 {
+    font-weight: normal;
+}
```

If you have identical rules for different elements, you can group them as so, when it makes sense:
```suggestion
h2,
h3 {
    font-weight: normal;
}
```